internal/service/clusteroutagesimulation/resource_cluster_outage_simulation_test.go

```diff
@@ -54,8 +61,15 @@ func TestAccOutageSimulationCluster_MultiRegion_basic(t *testing.T) {
 		CheckDestroy:             checkDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: configMultiRegion(projectName, orgID, clusterName),
+				Config: configMultiRegion(projectID, clusterName),
 				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "cluster_name", clusterName),
```

Comment:
Are these extra checks intentional?
Maybe worth refactoring the resource check into a helper function. Since these lines are duplicated across 4 test functions?

Reply:
i see pros in what you say, also the current approach is more verbose but more explicit, we can definitely explore your suggestion